v0/destinations/snapchat_conversion/transform.js

```diff
@@ -181,6 +179,10 @@ function trackResponseBuilder(message, { Config }) {
     );
   }
 
+  if (get(message, "properties.event_tag")) {
+    payload.event_tag = message?.properties?.event_tag;
```

Comment:
```suggestion
    payload.event_tag = message.properties.event_tag;
```
Since we are checking above optional chaining not required

Reply:
updated.